t/file.t

```diff
@@ -23,20 +23,21 @@ if ( $^O =~ m/^MSWin/ ) {
   $tmp_dir = $ENV{TMP};
 }
 
-my $filename = "$cwd/test-$$.txt";
+my $filename = "$cwd/test-{a,b}-$$.txt";
```

Comment:
That's true, but what if the simple filename handling and/or the array reference method breaks in the future, and the test suite only complains about a globbed form? I'd prefer making sure the test suite shows exactly what is broken to avoid extra debugging. Creating explicit test cases for the 3 different forms also has a second purpose as a form of documentation of intended/supported use cases.